packages/core/admin/ee/server/services/review-workflows/review-workflows.js

```diff
@@ -1,15 +1,21 @@
 'use strict';
 
-const { set, forEach, pipe, map } = require('lodash/fp');
+const { filter, set, forEach, pipe, map, stubTrue, cond } = require('lodash/fp');
 const { getService } = require('../../utils');
-const { getVisibleContentTypesUID } = require('../../utils/review-workflows');
+const { getVisibleContentTypesUID, hasStageAttribute } = require('../../utils/review-workflows');
 
 const defaultStages = require('../../constants/default-stages.json');
 const defaultWorkflow = require('../../constants/default-workflow.json');
 const { ENTITY_STAGE_ATTRIBUTE } = require('../../constants/workflows');
 
 const { persistTables, removePersistedTablesWithSuffix } = require('../../utils/persisted-tables');
 
+const MAX_DB_TABLE_NAME_LEN = 63; // Postgres limit
+// The suffix should looks like _strapi_reviewWorkflow_stage_links_inv_fk
```

Comment:
Maybe we could explain a bit why we use "_strapi_reviewWorkflow_stage_links_inv_fk" for the character limit.
```suggestion
// The longest index name that Strapi can create is prefixed with '_strapi_reviewWorkflow_stage_links_inv_fk', so the content type name  should be no longer than this.
```
wdyt?